vita3k/host/src/load_self.cpp

```diff
@@ -342,6 +342,11 @@ SceUID load_self(Ptr<const void> &entry_point, KernelState &kernel, MemState &me
         LOG_CRITICAL("SELF {} header type {} is not supported.", self_path, self_header.header_type);
         return -1;
     }
+    
+    if (self_path == "app0:sce_module/steroid.suprx") {
+        LOG_CRITICAL("You're trying to load a vitamin dump. It is not supported.");
```

Comment:
Let's do a popup if we can! [`error_dialog`](https://github.com/Vita3K/Vita3K/blob/master/vita3k/app/include/app/functions.h#L45) is good for this.

Reply:
Done.